src/buttons/buttonApply.command.js

```diff
@@ -0,0 +1,168 @@
+      emailFormName.val(ButtonApplyCommand.getDefaultEmailName());
+    }
+
+    Mautic.inBuilderSubmissionOn(emailForm);
+    Mautic.postForm(
+      emailForm,
+      ButtonApplyCommand.postFormResponse.bind(this, editor, sender, emailForm)
+    );
+    Mautic.inBuilderSubmissionOff();
+  }
+
+  /**
+   * Get and handle response
+   * Use the global Mautic functions
+   *
+   * @param editor
+   * @param sender
+   * @param emailForm
+   * @param response
+   */
+  static postFormResponse(editor, sender, emailForm, response) {
+    if (response.validationError !== null) {
+      const title = Mautic.translate('grapesjsbuilder.panelsViewsCommandModalTitleError');
+      ButtonApplyCommand.showModal(editor, title, response.validationError);
+    } else {
+      if (response.route) {
+        // update URL in address bar
+        MauticVars.manualStateChange = false;
+        History.pushState(null, 'Mautic', response.route);
+
+        // update Title
+        Mautic.generatePageTitle(response.route);
+      }
+
+      // update form action
+      if (ButtonApplyCommand.strcmp(emailForm[0].baseURI, emailForm[0].action) !== 0) {
+        emailForm[0].action = emailForm[0].baseURI;
+      }
+    }
+
+    sender.set('className', 'fa fa-check');
+  }
+
+  /**
+   * Create modal to show information about saving email
+   *
+   * @param editor
+   * @param title
+   * @param text
+   */
+  static showModal(editor, title, text) {
+    const modal = editor.Modal;
+    modal.setTitle(`<h4 class="text-danger">${title}</h4>`);
+
+    const body = document.createElement('div');
+    const content = document.createElement('div');
+    const footer = document.createElement('div');
+    const btnClose = document.createElement('button');
+
+    content.classList.add('panel-body');
+    content.innerText = text;
+    body.appendChild(content);
+
+    btnClose.classList.add('btn', 'btn-lg', 'btn-default', 'text-primary');
+    btnClose.innerText = 'Close';
+    btnClose.onclick = () => {
+      modal.close();
+    };
+    footer.classList.add('panel-footer', 'text-center');
+    footer.appendChild(btnClose);
+    body.appendChild(footer);
+
+    modal.setContent(body);
+    modal.open({
+      attributes: {
+        class: 'modal-content',
+      },
+    });
+  }
+
+  static getBtnViewsApply() {
+    return mQuery('#btn-views-apply');
+  }
+
+  static getEmailForm() {
+    return mQuery('form[name=emailform]');
+  }
+
+  static getEmailFormSubject(emailForm) {
+    return emailForm.find('#emailform_subject');
+  }
+
+  static getEmailFormName(emailForm) {
+    return emailForm.find('#emailform_name');
+  }
+
+  static getEmailFormList(emailForm) {
+    return emailForm.find('#emailform_lists');
+  }
+
+  static getEmailType(emailForm) {
+    return emailForm.find('#emailform_emailType');
+  }
+
+  static getDefaultEmailName() {
+    return `E-Mail ${moment().format('YYYY-MM-D hh:mm:ss')}`;
+  }
+
+  static strcmp(str1, str2) {
```

Comment:
@volha-pivavarchyk please add a description here too. so we know what this function is doing.